txnkv/transaction/txn.go

```diff
@@ -1704,6 +1704,10 @@ func (txn *KVTxn) StartTS() uint64 {
 	return txn.startTS
 }
 
+func (txn *KVTxn) CommitTS() uint64 {
```

Comment:
```suggestion
// CommitTS returns the commit timestamp of the already committed transaction, zero if it's not committed yet.
func (txn *KVTxn) CommitTS() uint64 {
```

Exported functions need explicity comments from the ci & lint.